<!--  for trajectory tracking -->
<launch>

    <arg name="mav_name" default="iris" />
    <arg name="namespace1" default="iris_leader" />
    <arg name="namespace2" default="iris_follower" />
    <!-- <rosparam command="load" file="$(find quadcopter_landing)/config/param.yaml" /> -->

    <group ns="$(arg namespace1)">
        <node name="lee_position_controller_node_weight" pkg="rotors_control" type="lee_position_controller_node_weight" output="screen">
            <rosparam command="load" file="$(find rotors_gazebo)/resource/lee_controller_$(arg mav_name).yaml" />
            <rosparam command="load" file="$(find rotors_gazebo)/resource/$(arg mav_name).yaml" />
            <remap from="odometry" to="odometry_sensor1/odometry" />
        </node>
        <!-- <node name="pub_position" pkg="quadcopter_landing" type="pub_position" output="screen" /> -->
        <!-- <node name="waypoint_publisher" pkg="quadcopter_landing" type="waypoint_publisher_file" output="screen" args="$(find quadcopter_landing)/config/waypoints.txt" /> -->
        <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
        <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
    </group>
    <!--
    <group ns="$(arg namespace2)">
        <node name="lee_position_controller_node" pkg="rotors_control" type="lee_position_controller_node" output="screen" >
            <rosparam command="load" file="$(find rotors_gazebo)/resource/lee_controller_$(arg mav_name).yaml" />
            <rosparam command="load" file="$(find rotors_gazebo)/resource/$(arg mav_name).yaml" />
            <remap from="odometry" to="odometry_sensor1/odometry" />
        </node>
        <node name="waypoint_publisher" pkg="quadcopter_landing" type="waypoint_publisher" output="screen" args="17.2 19 5 0 2" />
        <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
        <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
    </group>
    -->
    <!-- <node name="geo" pkg="offb" type="geo" output="screen"/> -->

</launch>
